participants/_template.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "My Name",
+  "company": "my company",
+  "tags": [],
+  "when": {
+    "saturday": true,
+    "sunday": false
+  },
+  "what_is_my_connection_to_javascript": "...",
+  "what_can_i_contribute": "???",
+  "urls": {
+    "photo": "",
+    "homepage": "",
+    "github": "",
+    "twitter": ""
+  }
+}
```

Comment:
If the t-shirt size is no longer needed, please also remove it from the registration page.